src/Result.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Exercism\PhpTestRunner;
+
+use JsonSerializable;
+
+final class Result implements JsonSerializable
+{
+    public function __construct(
+        private readonly string $testPrettyName,
+        private readonly string $testStatus,
+        private readonly string $testCode,
+        private int $taskId = 0,
```

Comment:
Should this be nullable?

Reply:
I am not so much of a `null` person. I prefer ["special cases"](https://martinfowler.com/eaaCatalog/specialCase.html) to handle the absence of a value, instead of requiring `null` checking / type conversion all over the code.